layer/logged_data/queuing_logged_data_destination.py

```diff
@@ -40,12 +41,14 @@ def receive(
         self,
         func: Callable[[LoggedDataClient], Optional[Any]],
         data: Optional[Any] = None,
+        data_path: Optional[pathlib.Path] = None,
     ) -> None:
```

Comment:
Maybe throw if both `data` and `data_path` are given?

Reply:
Done!